src/bindings/CMakeLists.txt

```diff
@@ -34,3 +34,10 @@ if (FINDEX GREATER -1)
 		add_subdirectory (gi)
 	endif ()
 endif ()
+
+list (FIND BINDINGS "gsettings" FINDEX)
```

Comment:
I thought gsettings uses glib binding of Elektra? If this is the case, the inclusion should be in the if block above (like the gi bindings)